.github/workflows/auto-update.yml

```diff
@@ -56,16 +55,46 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.BOT_TOKEN }}
         run: |
-          BRANCH_NAME=${BASE_BRANCH_NAME}-update-external-images
-          echo "MSG=Update module chart to ${TAG}" >> $GITHUB_ENV
-          prs=$(gh pr list -A $GIT_NAME --state open --json headRefName | jq -r '.[] | .headRefName')
-          if echo $prs | tr " " '\n' | grep -F -q -x $BRANCH_NAME; then
+          CHART_UPDATE_BRANCH_NAME=${BASE_BRANCH_NAME}-update-chart-and-external-images
+          echo "CHART_UPDATE_BRANCH_NAME=${CHART_UPDATE_BRANCH_NAME}" >> $GITHUB_ENV
+          MSG=Update module chart to ${TAG}
+          PRS=$(gh pr list -A $GIT_NAME --state open --json headRefName | jq -r '.[] | .headRefName')
+          if echo $PRS | tr " " '\n' | grep -F -q -x ${CHART_UPDATE_BRANCH_NAME}; then
             echo "PR already exists, no need to create a new one"
+            echo "CHART_UPDATE_PR_NUMBER=$(gh pr list --search "base:main head:${CHART_UPDATE_BRANCH_NAME}" --json number | jq -r '.[] | .number')" >> $GITHUB_ENV
           elif [ -z "$(git status --porcelain)" ]; then
-            echo "nothing changed, no need to create PR"
+            echo "Nothing changed, no need to create PR"
+            echo "CHART_UPDATE_PR_NUMBER=-1" >> $GITHUB_ENV
           else
-            scripts/update/create_pr_module_chart.sh
-          fi      
+            PR_STATUS=$(scripts/update/create_pr_module_chart.sh)
```

Comment:
Why in this PR we merge the image-syncer together with the module-chart? Can't the module-chart be merged in the next PR?

Reply:
It could be merged in the next PR, but I just see the chart update and the image sync as coupled process since the external-images.yaml is updated accordingly to the images referenced in the chart. Otherwise, the image-syncer will upload the images to the registry while there is a possibility to change them in the chart.